src/components/link/README.md

```diff
@@ -1 +1,44 @@
-# Link
\ No newline at end of file
+# Link
+
+Inline text links are used to navigate between documents (pages).
+
+## When to use
+
+- Navigating to a new page or view in your application
+- Navigating to different web page, e.g. external documentation
+
+## When to use something else
+
+Use the button component for:
+- Opening or closing a modal or dialog
+- Triggering a dropdown menu
+- Submitting data to the server
+
+## Adding the markup
+
+To mark up a link in Rivet, add the `a` HTML element.
```

Comment:
Should we be encouraging people to use just `a` tags, or `a` tags with the class `rvt-link`? I know we allow the use of either.

Reply:
Good catch! I've added a note and example using the `rvt-link` class. Let me know what you think!